includes/class-wp-job-manager-shortcodes.php

```diff
@@ -418,6 +418,10 @@ public function get_job_actions( $job ) {
 				break;
 			case 'pending_payment':
 			case 'pending':
+				$actions['continue'] = [
```

Comment:
We might need to split `pending` and `pending_payment` here. `pending` is if they've actually submitted it but `pending_payment` is before the process. `Edit` (when available based on setting) makes more sense for `pending` and `pending_payment`. 

We should probably put this in an action under `default:`, but somehow not break compatibility with older WCPL that don't hook into the new action.

Reply:
Yes I agree, and IMO it should set the status back to Draft to actually have the listing go through the status transitions when they actually occur.  Reason being the listing should go into `preview` status again, then transition to `pending_payment` (instead of just remaining on `pending_payment` through the entire flow)